File: httpsnippet/src/test/resources/output/node/request/application-form-encoded.js
var request = require("request");

var options = {"headers":{"content-type":"application/x-www-form-urlencoded"},"method":"POST","url":"http://mockbin.com/har","forms":{"foo":"bar","hello":"world"}};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: httpsnippet/src/test/resources/output/clojure/clj_http/application-form-encoded.clj
(require '[clj-http.client :as client])

(client/post "http://mockbin.com/har" {headers {content-type "application/x-www-form-urlencoded" 
                                         } 
                                      :form-params {foo "bar" 
                                          :hello "world" 
                                          } 
                                      })